src/handlers/notification/index.js

```diff
@@ -74,21 +74,17 @@ const processMessage = async (msg) => {
     throw new Error('Invalid message received from kafka')
   }
   const { metadata, from, to, content } = msg.value
-  const { action, status } = metadata.event
-  if (action === 'prepare' && status === 'success') {
+  const {status} = metadata.event
+  if (status === 'success') {
```

Comment:
We have different rules to handle notifications based on action. 
   - Prepare, Success - Send POST /transfers to Payee FSP
   - Fulfill Success - Send PUT /transfers/{ID} notification to both Payer and Payee FSP.

So I think we do need check for action and status in the if statement.